profiles/standard/standard.install

```diff
@@ -124,6 +124,23 @@ function standard_install() {
     node_add_body_field($type);
   }
 
+  $node = array(
+    'title' => 'Your first article!',
+    'body' => array(
+      LANGUAGE_NONE => array(
+        array(
+          'value' => st('This is your first article, edit it or delete it.'),
+        ),
+      ),
+    ),
+    'uid' => 1,
+    'status' => 1,
+    'promote' => 1,
+    'type' => 'article',
+  );
+  //$node = node_submit(entity_create('node', $node));
+  $node->save();
```

Comment:
You could also try `$node = new Node(array(...`

`$node->save()` won't work unless it's already a node object. You can use either `entity_create()` or just create the Node object directly.